packages/sui-domain/src/index.js

```diff
@@ -0,0 +1,24 @@
+
+import required from './helpers/required'
+
+export default class EntryPoint {
+  constructor ({ config = required('config'), useCases = require('useCases') }) {
+    this._config = config
+    this._useCases = useCases
+  }
+
+  get (key) {
+    if (key === 'config') {
+      return this._config
+    }
+
+    return this._useCases[key] || {
```

Comment:
Why are we informing of the absence of the use case only on "execute" call?
It's very weird to me to return an actual object and not be able to find out if the use case is present on the "domin.get()" function call...
That's a homemade pattern and I don't get what is it for.

Reply:
I understand that this would keep your code working safely but breaking in the `get` won't as you're trying to chain the `get` method. I copied it as we're doing in the actual domains and might be @carlosvillu has a better explanation.

Reply:
Chequear si hacer el get del key esta es `undefined`.